lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -1330,14 +1336,18 @@ class LineTouchData extends FlTouchData with EquatableMixin {
     LineTouchTooltipData? touchTooltipData,
     GetTouchedSpotIndicator? getTouchedSpotIndicator,
     double? touchSpotThreshold,
-    bool? fullHeightTouchLine,
     bool? handleBuiltInTouches,
+    GetTouchLineY? getTouchLineStart,
+    GetTouchLineY? getTouchLineEnd,
+    bool? touchLineEndAtDot,
```

Comment:
I suggest `preventOverlapWithDot` instead of `touchLineEndAtDot`.
(The functionality must be negated)

Reply:
BTW It has a lot of edge-cases.
For example, determine user getTouchLineStart start to -double.infinity, and getTouchLineEnd to +double.infinity while this flag is true.
And we decrease the end of the line by circle size which it passed over the circle.

Reply:
Please remove this flag, and if you want we can talk about it in a separate PR.
Because there are a lot of edge-cases that I can imagine.
Thanks!

Reply:
Sorry for the delay. I was out a few days.

Yeah, I think what you said makes sense. Removed this parameter. Though I would like to keep the current behavior that the line end doesn't overlap with the dot. Now, this is done by checking whether the line ends within the dot.

Reply:
No problem.
Great idea. Thanks, I'm going to merge it.